rts/ExternalAI/SSkirmishAICallbackImpl.cpp

```diff
@@ -1500,141 +1500,65 @@ EXPORT(int) skirmishAiCallback_Game_getTeamAllyTeam(int skirmishAIId, int otherT
 	return teamHandler.AllyTeam(otherTeamId);
 }
 
+static inline float getResourceFromPack(int resourceID, const SResourcePack& pack, float defaultValue = 0.0f)
+{
+	if (!resourceHandler->IsValidId(resourceID))
+		return defaultValue;
 
+	return pack[resourceID];
+}
 
-EXPORT(float) skirmishAiCallback_Game_getTeamResourceCurrent(int skirmishAIId, int otherTeamId, int resourceId)
+static inline float aiGetTeamResource(int skirmishAiID, int otherTeamID, int resourceID, SResourcePack CTeam::* resourcePack)
```

Comment:
This is probably the only non-trivial part so perhaps I could split it off if it makes review troublesome.